Dockerfile

```diff
@@ -75,6 +75,7 @@ RUN gem install \
 
 # Install Python packages
 RUN pip3 install \
+    authlib \
```

Comment:
@dmalan should we specify a min and a max version for that and other packages?

Reply:
https://github.com/cs50/cli/pull/89/files ?